modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -261,6 +272,21 @@
     /** Caches configured on local node but not presented in a cluster during node local join. */
     private volatile List<CacheData> onlyLocallyConfiguredCaches = Collections.emptyList();
 
+    /** Metastorage. */
+    private volatile ReadWriteMetastorage metastorage;
+
+    /** Metastorage synchronization mutex. */
+    private final Object metaStorageMux = new Object();
+
+    /** Set of workers that executing local continuous tasks. */
+    private final Set<GridWorker> asyncLocalContinuousTaskWorkers = new GridConcurrentHashSet<>();
+
+    /** Count of workers that executing local continuous tasks. */
+    private final AtomicInteger asyncLocalContinuousTasksWorkersCnt = new AtomicInteger(0);
```

Comment:
may be asyncLocalContinuousTaskWorkers.size() would be enough?

Reply:
Not sure about it - it's monotonic counter for worker naming. I remaned it to asyncLocalContinuousTasksWorkersCntr.